doc/rados/operations/health-checks.rst

```diff
@@ -1048,7 +1048,10 @@ PG_NOT_SCRUBBED
 _______________
 
 One or more PGs has not been scrubbed recently.  PGs are normally scrubbed
-within every ``osd_scrub_max_interval``, and this warning triggers when
+within every configured interval which is specified
+:ref:`osd_scrub_max_interval <osd_scrub_max_interval>` globally. And this
```

Comment:
I'm not a fan of conjunctions at the start of a sentence so you could either do 'globally, and this' or 'globally. This'?